docs/vgclog_examples.md

```diff
@@ -0,0 +1,595 @@
+|-------------|--------------|-------------------------------|--------------------------------------|---------------------------------------------------------------------------|
+|Scavenge     |initial        | STW              | `<gc-start>`, `<gc-end>`|Root scanning, reported as a single scavenge operation |
+|Scavenge     |intermediate        | concurrent         | `<concurrent-start>`, `<concurrent-end>`|Live objects are traversed and evacuated (copy-forwarded). Operation is reported as a `scavenge` operation |
+|Scavenge     |final     | STW              | `<gc-start>`, `<gc-end>` |weak roots scanning, reported as a complex scavenge operation (`<gc-op>`) containing specific details for each of weak root groups  |
+
+To search for a concurrent scavenge partial GC cycle, you can search for the `type` attribute value `scavenge` in `cycle-start` and `cycle-end` elements, or search for the `<af>` element that logs the allocation failure trigger. 
+
+You can locate the concurrent scavenge partial cycle's concurrent increment by searching for `<concurrent-start>` and `<concurrent-end>`. The global cycle's intermediate concurrent increment, which can run at the same time, is not logged by an element, but begins immediately after application threads are restarted following the `<cycle-start type="global"/>` element. For more information about the global cycle's intermediate concurrent increment, see [Log examples - Global GC Cycle](vgclog_examples.md#global-gc-cycle). For more information about GC increments, see [GC increments and interleaving](vgclog.md#gc-increments-and-interleaving).
+
+
+### Global GC cycle
+
+The following example shows how a global GC cycle is recorded in a `gencon` policy verbose GC log. The output is broken down into sections with supporting text to explain the GC processing that is taking place.
+
+The global GC cycle runs when the *tenure* area is close to full, which typically occurs after many partial cycles. As such, the output can be found part way down the full log. For more information about the GC Initialization section, see [Verbose GC log contents and structure ](vgclog.md#verbose-gc-log-contents-and-structure). For an example log output for a `gencon` partial cycle, see [Example - `gencon`’s default partial GC cycle](vgclog.md/#example-gencons-default-partial-gc-cycle).
+
+ [The global GC cycle is split into three increments](./vgclog.md#gc-increments-and-interleaving). Splitting the cycle operations into the following increments reduces pause times by running the majority of the GC work concurrently. The concurrent increment pauses when a partial GC cycle is triggered and resumes after the partial cycle, or multiple cycles, finish. The interleaving of partial GC cycles with the global cycle's intermediate concurrent increment can be seen in the following example. A single partial GC cycle is logged between the initial and final increments of the global cycle.
```

Comment:
I don't think we should link this whole sentence in this way for accessibility. We should say something like:

" The global GC cycle is split into three increments, as shown in [GC increments and interleaving]()."

Later on in this paragraph, we refer to the following example, but there isn't one. Is that one of the "to be done later" bits? If so please comment out.

Reply:
First point: resolved.

Second point: 'the following example' refers to the whol gencon global cycle section.

Amnended to: _The interleaving of partial GC cycles with the global cycle's intermediate concurrent increment can be seen in the following `gencon` global GC cycle log output._